config/jsdoc/api/plugins/doc.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Define an @doc tag
+ */
+var conf = env.conf.stability;
+var defaultLevels = ["deprecated","experimental","unstable","stable","frozen","locked"];
```

Comment:
I suppose there is the question of what stable/unstable means in the case of typedefs